app/types/redux.ts

```diff
@@ -27,6 +27,8 @@ export interface NodeState {
   version: string;
   build: string;
   error: NodeError | null;
+  filePermissionError: string | null;
```

Comment:
1. Let's move it to another place. It is not related to the node. For example into `UiState`. (Btw, there is already exists `error` prop which shows an error modal. But it is used mostly for uncaught exceptions).
2. I'm pretty sure it should be one field. Think about when we'll have more possible errors to notify about. E.g. `errors: MuCustomErrors[]`